dlt/cli/pipeline_command.py

```diff
@@ -134,3 +136,14 @@ def pipeline_command(operation: str, pipeline_name: str, pipelines_dir: str, ver
             fmt.echo("Found schema with name %s" % fmt.bold(p.default_schema_name))
         schema_str = p.default_schema.to_pretty_yaml(remove_defaults=True)
         fmt.echo(schema_str)
+
+    if operation == "drop":
+        drop = DropCommand(p, **drop_args)
+        if drop.drop_all:
+            fmt.echo("About to drop all destination and local data for the pipeline:")
```

Comment:
Would need destination name too, right? Shouldn't we add this to the sync command also and have it as a general util? Maybe an option in `dlt.attach` or another `dlt.attach_from_destination`

Reply:
@rudolfix I added this here for all the pipeline commands https://github.com/dlt-hub/dlt/pull/285/commits/942a52a954468e790e6886b6edfbdf6174375962  
When regular `attach` fails it will prompt and try to attach new pipeline and sync   

Let me know if you think it makes sense like this and I'll add tests, etc.

Reply:
ohhh I missed this comment. but my last PR answers it